src/Faker/Provider/ru_RU/Person.php

```diff
@@ -154,4 +156,25 @@ public function middleName($gender = null)
             static::GENDER_FEMALE,
         )));
     }
+
+    /**
+     * Return last name for the specified gender.
+     *
+     * @access public
+     * @param string|null $gender A gender of the last name should be generated
+     *     for. If the argument is skipped a random gender will be used.
+     * @return string Last name
+     */
+    public function lastName($gender = null)
+    {
+        $lastName = static::randomElement(static::$lastName);
+
+        if (static::GENDER_FEMALE === $gender) {
+            return $lastName . 'a';
+        } elseif (static::GENDER_MALE === $gender) {
+            return $lastName;
+        }
+
+        return $lastName . static::randomElement(static::$lastNameSuffix);
```

Comment:
What do you think about

```suggestion
        return static::lastName(static::randomElement(array(
            static::GENDER_FEMALE,
            static::GENDER_MALE,
        ));
```

?

Then we can also do away with the newly introduced `protected` property.